src/main/java/com/exactpro/th2/inframgr/k8s/SecretsManager.java

```diff
@@ -113,7 +114,7 @@ public Set<String> deleteSecrets(String schemaName, Set<String> secretEntries) {
         }
         secret.setData(data);
         try {
-            kubernetesClient.resource(secret).inNamespace(namespace).createOrReplace();
+            kubernetesClient.resource(secret).inNamespace(namespace).serverSideApply();
```

Comment:
Why have you started using `serverSideApply` instead `createOrReplace`? Is there any particular reason for that?

Reply:
createOrReplace is deprecated, auto-replacing recommends serverSideApply